packages/components/src/components/popover/popover.tsx

```diff
@@ -64,26 +68,22 @@ export class Popover {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
-    attachComponentCss(this.host, getComponentCss, this.direction);
+    attachComponentCss(this.host, getComponentCss, this.direction, this.theme);
 
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
       <Host onKeydown={this.onKeydown}>
-        <PrefixedTagNames.pButtonPure
+        <button
           type="button"
-          icon="information"
-          hideLabel={true}
           onClick={() => (this.open = !this.open)}
-          // pass string to avoid another update on p-button on each render because of new object reference
-          aria={JSON.stringify({
-            'aria-expanded': this.open,
-            ...parseAndGetAriaAttributes(this.aria),
-          })}
+          aria-expanded={this.open}
```

Comment:
I think this isn't the same as before since this is a boolean, not a string value. But merging this into the parameter of `parseAndGetAriaAttributes()` should take care of it.

Reply:
`this.aria` could be a string like '{ 'aria-label': 'Some label' }', so i dont know how we should merge this? I changed the value so aria-expanded is 'true' or 'false'

Reply:
Could be parsed to an object?